pkg/kubelet/dockershim/exec_test.go

```diff
@@ -145,7 +145,6 @@ func TestExecInContainer(t *testing.T) {
 	for _, tc := range testcases {
 		tc := tc
 		t.Run(tc.description, func(t *testing.T) {
-			t.Parallel()
```

Comment:
If we're going to un-parallelize this then we should remove the `tc := tc` statement on L146 above

Reply:
Also @pacoxu can you add a comment here that these tests *cannot* be run in parallel due to the fact that they are feature gate-dependent, and the `defer featuregatetesting.SetFeatureGateDuringTest` pattern breaks when tests are run in parallel

Good catch, sorry for this change making your life more annoying over the weekend!

Reply:
Updated.

Reply:
`tc := tc` is needed for preventing data race problem.

>     testing.go:1093: race detected during execution of test